File: Java/Fibonacci.java

import java.util.*;
class Main {
    public static void main(String[] args) {
        int i=0;
        int j=1;
        System.out.println("The fibonnaci series will be: ");
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter a number till which you want the fibonacci series...");
        int n=sc.nextInt();
        System.out.println("The Fibonacci series will be: ");
        for(int v=0;v<=n;v++)
        {
            System.out.print(i+" ");
            int m=i+j;
            i=j;
            j=m;
        }
    }
}


//output:
The fibonnaci series will be: 
Enter a number till which you want the fibonacci series...
5
    
0 
1 
1 
2 
3 
5 

File: Git_Demo/Fibonacci.py
The fibonacci series logic -->next=First+sec, first-sec, sec=next
same logic just syntax change

def fibonacci(n,first,sec):
    print("The Fibonacci series is:")
    for i in range(n):
        print("first",end=" ")
        next=first+sec
        first=sec
        sec=next

n=int(input("Enter the number till you want fibonacci series: ")) 
first=0
sec=1
fibonacci(n,first,sec)



//output:
Enter the number till you want fibonacci series: 20
The Fibonacci series is:
0 1 1 2 3 5 8 13 21 34 55 89 144 233 377 610 987 1597 2584 4181
